spec/02-integration/05-proxy/06-ssl_spec.lua

```diff
@@ -63,17 +62,6 @@ end
 
 
 local function gen_route(flavor, r)
-  if flavor ~= "expressions" then
-    return r
-  end
-
-  r.expression = atc_compat.get_expression(r)
-  r.priority = tonumber(atc_compat._get_priority(r))
-
-  r.hosts = nil
-  r.paths = nil
-  r.snis  = nil
-
   return r
 end
```

Comment:
Shall we remove this function?

Reply:
I think that we could do it but may not in this PR, we need some time to verify the effect of this PR. When we confirm it is not needed we will remove it.

Reply:
make sense